alma/sfcc-cartridge
# Add test warning on Frontstore

## Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1933/[-sfcc]-warn-user-that-module-is-in-test-mode)

### Code changes


Add a param in plans formatted for checkout and if we are in test mode, add ma warning in the checkout.

### How to test

_As a reviewer, you are encouraged to test the PR locally._
Go to the Storefront in test mode and see the warning on each Alma payment method.



### Checklist for authors and reviewers




### Non applicable
